tests/parallel/tbetterSpawnErrorMessage.nim

```diff
@@ -1,6 +1,7 @@
 discard """
   matrix: "--threads:on"
   errormsg: "'spawn' takes a call expression; got: proc (x: uint32) = echo [x]"
+  description: "Poor error message for spawn when a procedure (without calling it) is passed, issues #7535"
```

Comment:
Good news the parallel category got murdered in `devel`, so you don't have to worry about that one. :D